package.json

```diff
@@ -72,28 +72,28 @@
     "watchify": "^3.7.0"
   },
   "scripts": {
-    "clean": "rm -rf ./dist ./node_modules ./examples/build.dev.js ./examples/build.prod.js",
-    "disc": "npm-run-all dist disc:build disc:open",
-    "disc:build": "mkdir -p ./tmp && browserify ./dist/index.js --full-paths > ./tmp/build.js",
+    "clean": "rm -rf ./lib ./node_modules ./examples/build.dev.js ./examples/build.prod.js",
+    "disc": "npm-run-all build disc:build disc:open",
+    "disc:build": "mkdir -p ./tmp && browserify ./lib/index.js --full-paths > ./tmp/build.js",
     "disc:open": "discify ./tmp/build.js --open",
-    "dist": "npm-run-all dist:npm dist:max dist:min",
-    "dist:max": "NODE_ENV=production browserify ./lib/index.js --transform babelify --transform envify --transform [ browserify-global-shim --global ] --standalone Slate > ./dist/slate.js",
-    "dist:min": "NODE_ENV=production browserify ./lib/index.js --transform babelify --transform envify --transform [ browserify-global-shim --global ] --transform uglifyify --standalone Slate | uglifyjs > ./dist/slate.min.js",
-    "dist:npm": "babel --out-dir ./dist ./lib",
+    "build": "npm-run-all build:npm build:max build:min",
+    "build:max": "NODE_ENV=production browserify ./src/index.js --transform babelify --transform envify --transform [ browserify-global-shim --global ] --standalone Slate > ./lib/slate.js",
```

Comment:
Yeah, I didn't really get that there where actual dist files compiled.. I've fixed this to target the dist folder. Also I've reconsidered un-ignoring the dist folder, as you brought up.